spec/content_tagger/removing_taxon_spec.rb

```diff
@@ -0,0 +1,37 @@
+feature "Removing content from Content Tagger", new: true, collections: true, content_tagger: true do
+  include ContentTaggerHelpers
+
+  let(:redirection_destination_title) { "Removed taxon destination " + SecureRandom.uuid }
+  let(:redirection_destination_slug) { "redirection-taxon-" + SecureRandom.uuid }
+  let(:removed_slug) { "removed-taxon-" + SecureRandom.uuid }
+
+  scenario "Unpublishing a taxon" do
+    given_there_are_two_published_taxons
+    and_i_remove_one_by_redirecting_to_the_other
+    then_visiting_the_removed_taxon_redirects_to_the_other_taxon
+  end
+
+  def given_there_are_two_published_taxons
+    create_draft_taxon(slug: redirection_destination_slug, title: redirection_destination_title)
+    publish_taxon
+    @redirection_destination_url = find_link("/" + redirection_destination_slug)[:href]
+    reload_url_until_status_code(@redirection_destination_url, 200)
```

Comment:
I wonder if we actually want/need to wait on this (and the subsequent one). We could instead just have the one wait and allow that to retry while 200 or 404?

Reply:
Previously we've made sure the initial item is published before trying to redirect to it.

Reply:
I'd imagine it depends on context, I can't see how it's adding something here. But maybe I'm missing something?